### ISSUE TITLE
Order Status

### ISSUE THREAD

**Issue by eyesuk** [alienbrett/PyAlly#21]
Is there a way that we can get a list of order status? 
According to the Api documentation [here](https://www.ally.com/api/invest/documentation/accounts-id-orders-get/) it appears that you can using the URL you are using in the order_history, however I am unable to retrieve a pending order status.  Am I looking at this incorrectly?

Thanks for your help.

**Comment by JTraversa** [comment]
I'm still waiting on funds to transfer so cannot test personally to make sure my solution would work. So i'll return to this thread once my funds have cleared.

But in the meantime / if it does not work, you could tie in get_accounts() or get_holdings and use cash balances + holdings to deduce order status.

**Comment by eyesuk** [comment]
Thank you man!

**Comment by alienbrett** [comment]
I actually have the same problem as you eyesuk. I've tried to gauge recent order information using that function and it conforms to ally's specifications, but I've never got any valid information from the api endpoint. I'm in communication with the API team, and I'll bring this up.

**Comment by alienbrett** [comment]
So the new endpoint in V1.0.1 returns all order placed within the last 24 hours. Please open a new issue if this is still a problem with the latest version
